pom.xml

```diff
@@ -606,6 +606,14 @@ The following provides more details on the included cryptographic software:
           <ignoredDifferencesFile>${basedir}/clirr-excludes.xml</ignoredDifferencesFile>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-release-plugin</artifactId>
+	      <version>2.5.3</version>
+	      <configuration>
+          <preparationGoals>verify</preparationGoals>
+	      </configuration>
```

Comment:
Indentation is funky.

Reply:
My text editor for some reason is mixing tabs and spaces.  Should be fixed now.